concent_api/gatekeeper/views.py

```diff
@@ -0,0 +1,68 @@
+from base64                         import b64decode
+
+from django.conf                    import settings
+from django.http                    import HttpResponse
+from django.views.decorators.csrf   import csrf_exempt
+from django.views.decorators.http   import require_POST
+from django.views.decorators.http   import require_GET
+
+from gatekeeper.utils               import gatekeeper_access_denied_response
+
+from golem_messages.message         import init_messages
+from golem_messages.message         import Message
+from golem_messages.shortcuts       import load
+
+
+@csrf_exempt
+@require_POST
+def upload(request):
+    if request.content_type != 'multipart/form-data':
+        return gatekeeper_access_denied_response('Wrong upload data content type.')
+    init_messages()
+    check_header = check_headers(request)
+    if check_header is not None:
+        return check_header
+
+    return HttpResponse("", status=200)
+
+
+@csrf_exempt
+@require_GET
+def download(request):
+    if request.content_type == 'multipart/form-data':
+        return gatekeeper_access_denied_response('Wrong download data content type.')
+    init_messages()
+    check_header = check_headers(request)
+    if check_header is not None:
+        return check_header
+
+    return HttpResponse("", status=200)
+
+
+def check_headers(request):
+    # Decode and check if request header contains golem message:
+    if 'HTTP_AUTHORIZATION' not in request.META:
+        return gatekeeper_access_denied_response('Missing Authorization header.')
+
+    authorization = request.META['HTTP_AUTHORIZATION']
+
+    if len(authorization.split(" ")) != 2:
+        return gatekeeper_access_denied_response('Wrong construction of Authorization header.')
+    (golem_header, auth_header_content) = authorization.split()
```

Comment:
() is not needed here.

Reply:
Right! Fixed.

Reply:
About these parentheses. They're optional but I encourage using them because they make it actually look like a tuple. With them it's easier to see that you're assigning multiple values there.

Same with return values. `return 1, 2` is valid but for me `return (1, 2)` is more readable because you immediately see that you're returning a tuple of two values.